internal/sirius/client.go

```diff
@@ -85,3 +89,11 @@ func (c *Client) newRequest(ctx Context, method, path string, body io.Reader) (*
 
 	return req, err
 }
+
+func getContext(cookies []*http.Cookie) Context {
+	return Context{
+		Context:   context.Background(),
+		Cookies:   cookies,
+		XSRFToken: "abcde",
```

Comment:
Is this XSRToken correct? It look strange

Reply:
You're right, not sure what I did there